flake.nix

```diff
@@ -21,6 +21,7 @@
 
   outputs = { self, nixpkgs, nix, ... }@inputs:
     let
+      mncMeta = (builtins.fromTOML (builtins.readFile ./magic-nix-cache/Cargo.toml)).package;
```

Comment:
Any reason why this has to be in the flake and not in the `package.nix` anyways? It would at least allow a smaller diff (not that that's worth optimizing for if you don't care)

Reply:
No specific reason. Will change.